src/d3a/models/market/blockchain_interface.py

```diff
@@ -199,12 +97,21 @@ def handle_blockchain_trade_event(self, offer, buyer, original_offer, residual_o
         return str(uuid.uuid4()), residual_offer
 
     def track_trade_event(self, trade):
-        path = Path.cwd().joinpath('src', 'd3a', 'models', 'market', 'metadata.json')
-        trade_id, call = self.storage_contract_call(
-            self.contracts['main'],
-            "trade",
-            str(path)
+
+        call_params = {
+            'id': 3,
+            'buyer': ALICE_STASH_ADDRESS,
+            'seller': BOB_STASH_ADDRESS,
+            'amount': default_amount,
```

Comment:
Its not helpful to always document trade with a default_amount. I would rather vouch for to document an actual traded energy here

Reply:
For the sake of simplicity, I did all defaults to make sure the blockchain stuff works. Should we do this in another ticket?

Reply:
sounds good, please add a ticket to deal with it later.

Reply:
https://gridsingularity.atlassian.net/secure/RapidBoard.jspa?rapidView=43&projectKey=DD&view=detail&selectedIssue=DD-14